corehq/apps/userreports/reports/specs.py

```diff
@@ -333,6 +342,17 @@ class ExpressionColumn(BaseReportColumn):
     def wrapped_expression(self):
         return ExpressionFactory.from_spec(self.expression)
 
+    def get_column_config(self, data_source_config, lang):
+        return ColumnConfig(columns=[
+            CalculatedColumn(
+                header=self.get_header(lang),
+                slug=self.column_id,
+                # todo: are these needed?
+                # format_fn=self.get_format_fn(),
```

Comment:
I think that you could have the `wrapped_expression` called in the `format_fn`. I think it gets called somewhere deep down in reports in `report_column.format_data()`. The exact syntax/structure you would need escapes me at the moment, but percent columns use this, to combine two columns which I think is what this is doing